src/containers/flows/AcceptBid/AcceptBid.tsx

```diff
@@ -43,12 +43,12 @@ type AcceptBidProps = {
 	assetUrl: string;
 	creatorId: string;
 	domainTitle: string;
-	domainId: string;
 	domainName: string;
 	walletAddress: string;
 	highestBid?: string;
 	onClose: () => void;
 	paymentTokenInfo: ConvertedTokenInfo;
+	setIsAcceptBidProcessing: (state: boolean) => void;
```

Comment:
Could be beneficial to make this one more generic, perhaps it could be a callback which is fired for each step of the AcceptBid flow.

`onChangeStep: (step: Step) => void`

```js
const changeStep = (step: Step) => {
    onChangeStep(step);
    setCurrentStep(step);
}
...
const onAcceptBid = () => {
    changeStep(Step.Processing);
    try {
        await accept(acceptingBid!);
        changeStep(Step.Success);
    } catch (e) {
        changeStep(Step.Failed);
    }
}
```

Or something like that?

Reply:
This is being used to handle disabled UI on for bid list, a change in step wouldn't be required. So if a user clicks out of the accept bid modal while transaction is pending, they will see: 

<img width="928" alt="Screenshot 2022-07-20 at 13 46 53" src="https://user-images.githubusercontent.com/39112648/179986188-4f348dc5-f7f7-4336-b243-8f9c236549c3.png">

Reply:
Gotcha makes sense!